webapp/utils/constants.jsx

```diff
@@ -894,6 +894,9 @@ export const Constants = {
     MIN_HASHTAG_LINK_LENGTH: 3,
     CHANNEL_SCROLL_ADJUSTMENT: 100,
     EMOJI_PATH: '/static/emoji',
+    CHANGE_EVENT: 'changed',
+    RECENT_EMOJI_KEY: 'recentEmojis',
+    MAXIMUM_RECENT_EMOJI: 27,
```

Comment:
Thanks @R-Wang97! If you have time to submit the update by early Tuesday, we should be able to take this one for the next Mattermost release (3.8)